packages/core/package.json

```diff
@@ -48,7 +48,8 @@
     "fix:eslint": "eslint . --format stylish --fix",
     "test": "jest",
     "test:watch": "jest --watch",
-    "pack": "npm pack"
+    "pack": "npm pack",
+    "version": "node ../../scripts/versionbump.js src/version.ts"
```

Comment:
Any reason not to get rid of the corresponding lines in the browser and node packages' respective `package.json` files?